docs/services/aws-services-guide.md

```diff
@@ -895,28 +459,20 @@ In this guide we:
 * Configured RDS to communicate with EKS
 * Installed Crossplane from the alpha channel
 * Installed the AWS stack
-* Created an infrastructure (`aws-infra-dev`) and application
-  (`app-project1-dev`) namespace
 * Setup an AWS `Provider` with our account
-* Created a `RDSInstanceClass` in the ` with configuration using RDS configuration earlier
-* Created a `MySQLInstanceClass` that specified the `RDSInstanceClass` as
-  `mysql-standard` in the `app-project1-dev` namespace
-* Created a `MySQLInstance` claim in the `app-project1-dev1` namespace that
-  referenced `mysql-standard`
+* Created a `RDSInstanceClass` with configuration for an AWS RDS instance
+* Created a `MySQLInstance` claim that was scheduled to the `mysql-standard`
```

Comment:
Nitpick: I've been saying "defaulted" when we use the annotation to distinguish it from using the class selector.